modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a single service and apply it to all of your projects. You can configure this type of environment by setting your Ingress Controller to use the `NodePortService` endpoint publishing strategy.
+
+Consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to a node even if the node gets displaced. 
+
+Before you `NodePort`-type `Service` for each project, read the following considerations:
+
+* You must create a wildcard DNS record for the Ingress Controller.
+* You must create a route service and specify the hostname of your Ingress Controller in this service.
+* must preserve the non-default port in the curl in order to access, for example: curl <route>:<port> // what does this mean?
+* Ensure that Ingress Controller routes can access your {product-title} cluster through the host IP address assigned to each node.
+
+.Prerequisites
+
+* You installed the {oc-first}.
+* Logged in as a user with `cluster-admin` privileges.
+* You created a wildcard DNS record.
+// https://docs.openshift.com/container-platform/4.16/networking/ingress-controller-dnsmgt.html (does not detail how to create the DNS)
+
+.Procedure
+
+. Create a custom resource (CR) file for the Ingress Controller:
++
+.Example of a CR file that defines information for the Ingress Controller
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: custom-ic3 <1>
```

Comment:
For items marked <1> to <4> can I replace the values with something the user needs to replace. For example:

* custom-ic3 = <ingress_controller_name>

Reply:
each one of these parameter is described in the table at https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/configuring-ingress#nw-ingress-controller-configuration-parameters_configuring-ingress.  The values in the .yaml file are valid as is.  In this case these are are things I made up:
1) name: custom-ic3 
2) domain:  my-custom-ic.ipi-cluster.example.com
3) ingresscontroller: custom-ic3 (nodePlacement label)
4) ic: custom-ic3 (namespaceSelector label)

Reply:
I replaced these values with user-variables but took info from the source table you linked. to describe the parameters.